app/api_client/base.py

```diff
@@ -34,11 +36,25 @@ def pretty_print_request(prep):
         ))
 
 
+class GenericRequester(object):
+    def __init__(self, client):
+        self.client = client
+
+    def __getattr__(self, name):
```

Comment:
Problem is the api client actually does more than just pass params in some cases, but work for the majority of calls